packages/vue/src/components/atoms/SfChevron/SfChevron.html

```diff
@@ -0,0 +1,6 @@
+<div class="sf-chevron" :style="getStyle">
+  <slot>
```

Comment:
```suggestion
  <slot v-bind="{isActive}">
```
@Michal-Dziedzinski nice to need when isActive inside slot :)